parts/inc/uv

```diff
@@ -103,14 +103,23 @@ my_strnlen(const char *str, Size_t maxlen)
  * intended to work with modern perls, the tighter restrictions could be
  * relaxed.  khw thinks this is unlikely, but has been wrong in the past. */
 
+#if { VERSION < 5.10.0 }
+# define cU8 U8
+#else
+# define cU8 const U8
+#endif
```

Comment:
Custom local/private defines not used outside of ppport.h should have `D_PPP_` prefix.
Therefore use:
```c
#if { VERSION < 5.10.0 }
#  define D_PPP_CU8 U8
#else
#  define D_PPP_CU8 const U8
#endif
```

Reply:
`D_PPP_CU8` is definitively a better choice and would avoid conflicts thanks for the update